test/cypress/integration/04-collective_v2.test.js

```diff
@@ -1,41 +1,86 @@
 import { disableSmoothScroll } from '../support/helpers';
 import { Sections } from '../../../components/collective-page/_constants';
+import 'cypress-file-upload';
 
-describe('New collective page', () => {
-  let collectiveSlug = null;
+const uploadImage = ({ dropzone, file }) => {
+  cy.fixture(`./images/${file}`).then(picture => {
+    cy.get(dropzone).upload({
+      fileContent: picture,
+      fileName: file,
+      mimeType: 'image/'.concat(file.includes('.png') ? 'png' : 'jpeg'),
+    });
+  });
+  cy.wait(900);
+};
 
-  const scrollToSection = section => {
-    // Wait for new collective page to load before disabling smooth scroll
-    cy.contains('Become a financial contributor');
-    disableSmoothScroll();
-    cy.get(`#section-${section}`).scrollIntoView();
-  };
+const scrollToSection = section => {
+  // Wait for new collective page to load before disabling smooth scroll
+  cy.contains('Become a financial contributor');
+  disableSmoothScroll();
+  cy.get(`#section-${section}`).scrollIntoView();
+};
 
+describe('New collective page', () => {
+  let collectiveSlug = null;
   before(() => {
-    cy.createHostedCollective()
+    cy.createHostedCollective({
+      twitterHandle: 'testCollective',
+      githubHandle: 'testCollective',
+      website: 'opencollective.com/testCollective',
+    })
       .then(({ slug }) => (collectiveSlug = slug))
       .then(() => cy.visit(`/${collectiveSlug}/v2`));
   });
 
+  beforeEach(() => {
+    cy.login({ redirect: `/${collectiveSlug}/v2` });
+    cy.wait(900);
+  });
+
   describe('Hero', () => {
-    it.skip('Must have links to twitter, github and website', () => {
-      // TODO
+    it('Must have links to twitter, github and website', () => {
+      cy.get('[data-cy=twitterProfileUrl]').should('have.attr', 'href', 'https://twitter.com/testCollective');
+      cy.get('[data-cy=githubProfileUrl]').should('have.attr', 'href', 'https://github.com/testCollective');
+      cy.get('[data-cy=collectiveWebsite]').should('have.attr', 'href', 'https://opencollective.com/testCollective');
     });
 
-    it.skip('Must have the fiscal host displayed', () => {
-      // TODO
+    it('Must have the fiscal host displayed', () => {
+      cy.get('[data-cy=fiscalHostName]').should('have.visible');
```

Comment:
Let's check that it properly shows the host name, here it should be something like `Open Source Collective`

`cy.get('[data-cy=fiscalHostName]').contains(______);`

Reply:
Understood